src/app/todo-card/todo-card.component.html

```diff
@@ -1,6 +1,5 @@
 <div class="todo-card">
   <h1 class="card-title">{{ todo.title }}</h1>
-  <p class="card-description">{{ todo.description }}</p>
```

Comment:
A description is completely removed, is this intentional?

Reply:
Yeah, it is redundant